Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -434,3 +434,52 @@ internal class SubjectAltNameSequence: Sequence, IteratorProtocol {
         CNIOBoringSSL_GENERAL_NAMES_free(self.nameStack)
     }
 }
+
+extension NIOSSLCertificate: CustomStringConvertible {
+    
+    public var description: String {
+        var desc = "<"
+        if let commonNameBytes = self.commonName() {
+            let commonName = String(decoding: commonNameBytes, as: Unicode.UTF8.self)
+            desc += ";common_name=" + commonName
+        }
+        if let alternativeName = self.subjectAlternativeNames() {
+            desc += ";alternative_names="
+            for altName in alternativeName {
+                switch altName {
+                case .dnsName(let bytes):
+                    desc += "\(String(decoding: bytes, as: Unicode.UTF8.self)),"
+                case .ipAddress(let address):
+                    desc += "\(ipToString(address)),"
+                }
+            }
+        }
+        return desc + ">"
+    }
+    
+    private func ipToString(_ address: IPAddress) -> String {
+        switch address{
+        case .ipv4(let addr):
+            return ipv4ToString(addr)
+        case .ipv6(let addr):
+            return ipv6ToString(addr)
+        }
+    }
+    
+    private func ipv4ToString(_ address: in_addr) -> String {
+        var address = address
+        let pointer = malloc(Int(INET_ADDRSTRLEN)).assumingMemoryBound(to: CChar.self)
```

Comment:
We can't use `assumingMemoryBound` like this. If we were to do this it'd be `bindMemory(to:)` but we may as well allocate the memory pre-bound. Additionally, let's remove `INET_ADDRSTRLEN`: it's not really helping us at all here, and it becomes an awkward symbol we have to remove down the road. Let's use a `let ipv4Len = 16`.

```suggestion
        let pointer = UnsafeMutableBufferPointer<CChar>.allocate(ipv4Len)
```